cli/cli/helpers/grafloki/docker_grafloki.go

```diff
@@ -107,7 +121,7 @@ func createGrafanaAndLokiContainers(ctx context.Context, dockerManager *docker_m
 		ApiVersion: int64(1),
 		Datasources: []GrafanaDatasource{
 			{
-				Name:      LokiContainerName,
+				Name:      LokiContainerNamePrefix,
```

Comment:
Think this is a bug - should be `lokiContainerName`, else it won't pick up any container